content/work/quarqnet.rst

```diff
@@ -48,3 +100,296 @@ summary: |
+as well as an automated "living" `style guide`_.
+
+.. _style guide: http://quarqnet.com/styleguide/config-colors.html
+
+.. figure:: /static/images/work/quarqnet/communication.jpg
+   :class: extend-large
+   :alt: user story in Pivotal Tracker and Sass color maps
+
+Initial Sketches
+~~~~~~~~~~~~~~~~
+
+.. image:: /static/images/work/quarqnet/activity.jpg
+   :class: extend-left img-shadow
+   :alt: initial sketch of activity map and data
+
+Because of our focus on users,
+OddBird's development process
+for each user story
+begins with rough sketches
+showing the flow of the user
+through the content.
+The athletes using QuarqNet
+needed easy access to the map of their route
+as well as the activity data.
+This initial design proposal
+displays the map and activity details together
+in a single view.
+No clicking necessary
+to access the most useful information
+on both mobile and larger screens.
+
+Hi-Fi Mockups
+~~~~~~~~~~~~~
+
+After the design direction
+for a user story is approved,
+we create static or interactive mockups
+often using Adobe Xd.
+These mockups allow us
+to experiment with brand implementation
+as well as clarify user flow and interaction -
+identifying sticking points
+and iterating quickly.
+In conversation with Quarq,
+it became clear that activity data needed a new layout
+to improve scannability.
+We don't linger too long on this step
+or attempt pixel perfection
+as new UX problems nearly always present themselves
+during implementation.
+
+.. image:: /static/images/work/quarqnet/mockup1.jpg
+   :class: extend-large
+   :alt: initial mockups of activity map and data
+
+Implementation
+~~~~~~~~~~~~~~
+
+Additional UX design iteration
+happens during implementation
+as we begin to interact with the new features
+in the context of the browser.
+Whether working with our own internal developers
+or partnering with external developers,
+we maintain a tight a feedback loop
+between designers and developers
+as well as the clients at this stage.
+We have frequent video calls
+to demo, discuss, pair, and re-design features.
+For this QuarqNet feature,
+we improved the dotted line, font size, and spacing
+of the activity data
+within the browser.
+
+.. image:: /static/images/work/quarqnet/implementation1.jpg
+   :class: extend-small img-shadow
+   :alt: initial implementation of activity map and data
+
+Bug Story
+~~~~~~~~~
+
+As we work our way
+through the list of user stories
+and test the implemented features with users,
+bugs and user flow issues are identified.
+The activity data box on QuarqNet
+became cluttered over time.
+We needed to use the same box
+to show both activity data,
+a list of past activities,
+and much more.
+During testing,
+users were not able to understand
+how to move quickly
+between the pieces of information they needed.
+A new Bug Story was born:
+
+*The multiple click targets
+around an activity
+(switch to list, hide/show details)
+are not clear enough.*
+
+Iteration - Sketches
```

Comment:
Start an iteration section with a second-level heading, rather than prefixing. I think this section also needs a bit of introduction before jumping right into "again..." – though overall I think these iteration steps can be shortened. We've already covered the flow of our process, so maybe don't have to break it into such small pieces for each iteration?

I also wonder if we need to frame the idea that iterations are a good thing. It looks a bit like we just got it wrong, and had to fix it. I don't think that's quite the story we're trying to tell.